examples/index.js

```diff
@@ -352,6 +352,14 @@ createPiles(exampleEl.value).then(([pilingLib, additionalOptions = []]) => {
       id: 'grid',
       title: 'Grid',
       fields: [
+        {
+          name: 'cellSize',
```

Comment:
Let's keep `itemSize` as the first property as it's most often adjusted I think :)